### PR TITLE
Fix tabulation

### PR DESCRIPTION
**Pull request by kgeorgiev77**

### CODE REVIEW

File: apply-submission.php [hunk 1]
```diff
@@ -1,107 +1,110 @@
...
+								<input type="text" placeholder="Last Name*" value="<?php echo $user["last_name"] ?>" />
+							</div>
+							<div class="form-field-wrapper width-medium">
+								<input type="text" placeholder="Email*" value="<?php echo $user["email"] ?>" />
+							</div>
+							<div class="form-field-wrapper width-medium">
+								<input type="text" placeholder="Phone Number" value="<?php echo $user["phone_number"] ?>" />
+							</div>			
+							<div class="form-field-wrapper width-large">
+								<textarea name="custom_message" placeholder="Custom Message*"></textarea>
```
Comment (vabrashev) [inline apply-submission.php:91]: Do we have the description when we are editing the job?

File: category-dashboard.php [hunk 2]
```diff
@@ -1,76 +1,76 @@
...
+	<section class="section-fullwidth section-jobs-dashboard">
+		<div class="row">						
+			<div class="jobs-dashboard-header">
+				<div class="primary-container">							
+					<ul class="tabs-menu">
+						<li class="menu-item">
+							<a href="dashboard.php">Jobs</a>					
+						</li>
+						<li class="menu-item current-menu-item">
+							<a href="#">Categories</a>
```
Comment (vabrashev) [inline category-dashboard.php:17]: Just to confirm we don't need the link for the Categories?

File: category-dashboard.php [hunk 3]
```diff
@@ -1,76 +1,76 @@
...
+							<div class="flex-container justified-vertically">									
+								<div class="form-field-wrapper">
+									<input name="title" type="text" placeholder="Enter Category Name..."/>
+								</div>
+									<button type="submit" class="button">
+										Add New
+									</button>		
+							</div>	
+							<?php 
+							require_once 'category_add.php';
```
Comment (vabrashev) [inline category-dashboard.php:33]: Should we have this in the middle of the HTML or it's possible to move it at the start of the file?

File: category_add.php [hunk 4]
```diff
@@ -1,30 +1,26 @@
...
-                                        VALUES ('".$title."')";       
-                        if ($con->query($sql_request) === TRUE) {
-                            echo '<div class="flex-container justified-vertically" style=color:#3c71fe>Category created</div>';
-                            } else {
-                                echo "ERROR: " . $sql_request . "<br>";
-                            }
-                        }
-                        
-                }
+if($_SERVER["REQUEST_METHOD"] == "POST"){
```
Comment (vabrashev) [inline category_add.php:2]: It's always a good thing to check what action exactly we want to perform and not only checking if it's POST or GET

File: dashboard.php [hunk 5]
```diff
@@ -1,152 +1,150 @@
...
+							<div class="search-form-field"> 
+								<input class="search-form-input" type="text" value="<?php if(!empty($_GET['search'])){echo $_GET['search'];}?>" placeholder="Search…" name="search" > 
+							</div> 
+						</div>
 						<?php 
+							if (!empty($_GET['drop_down_menu'])) {
+								$drop_down = $_GET['drop_down_menu'];
+							} 
+							else {
+								$drop_down = 1;
```
Comment (vabrashev) [inline dashboard.php:38]: We can move this over the if() check and just as a default value and then change it if $_GET['drop_down_menu'] is not empty

File: dashboard.php [hunk 6]
```diff
@@ -1,152 +1,150 @@
...
+					</form>
+				</div>
+			</div>
+			<ul class="jobs-listing">
+			<?php
+			if(isset($_GET['drop_down_menu']) && $_GET['drop_down_menu'] == 2){
+				$order_list = "jobs.title ASC";
+			} 
+			else{
+				$order_list = "jobs.date_posted DESC";
```
Comment (vabrashev) [inline dashboard.php:62]: We can move this over the if() check and just as a default value and then change it if $_GET['drop_down_menu'] is not empty

File: dashboard.php [hunk 7]
```diff
@@ -1,152 +1,150 @@
...
+				$result = mysqli_query($con, $sql); 
+			}
+			if(!empty($result)){		
+				while($row = mysqli_fetch_array($result, MYSQLI_ASSOC)) {	
+			?>
+				<li class="job-card">
+					<div class="job-primary">
+						<h2 class="job-title"><a href="single.php?id=<?php echo $row["id"]?>"><?php echo  $row["title"] ?></a></h2>
+						<div class="job-meta">
+							<a class="meta-company" href="single.php?id=<?php echo $row["id"]?>"><?php echo  $row["company_name"] ?></a>
```
Comment (vabrashev) [inline dashboard.php:91]: Is this link correct?

File: dashboard_delete.php [hunk 8]
```diff
@@ -1,17 +1,17 @@
...
-    }
+if(!empty($_POST['id'])){
+    $id = $_POST['id'];  
+    $sql_job = "DELETE FROM jobs WHERE jobs.id = $id";
+    $sql_categories = "DELETE FROM  jobs_categories WHERE  jobs_categories.job_id = $id";
+    $sql_submissions = "DELETE FROM  applications WHERE  applications.job_id = $id";
+    $res = $con->query($sql_job);
+    $res1 = $con->query($sql_categories);      
+    $res2 = $con->query($sql_submissions);  
+    echo $res;
```
Comment (vabrashev) [inline dashboard_delete.php:14]: We should check if all queries are successfully, but we are returning only the result for the first query.

File: single.php [hunk 9]
```diff
@@ -1,116 +1,115 @@
...
+		<div class="row">
+			<div class="job-single">
+				<div class="job-main">
+					<div class="job-card">
+						<div class="job-primary">
+							<header class="job-header">
+								<h2 class="job-title"><a href="#"><?php echo  $row["title"] ?></a></h2>
 								<div class="job-meta">
-									<a class="meta-company" href="single.php?id=<?php echo $row["id"]?>"><?php echo  $row["company_name"] ?></a>
+									<a class="meta-company" href="#"><?php echo  $row["company_name"] ?></a>
```
Comment (vabrashev) [inline single.php:21]: Where should Company Name lead us?

File: single.php [hunk 10]
```diff
@@ -1,116 +1,115 @@
...
+				<?php
+					$jobs = ShowRelJobs($row["id"]);
+					foreach($jobs as $row){
+						$image_path = IMAGE_PATH.$row["company_image"];
+				?>
+				<li class="job-card">
+					<div class="job-primary">
+						<h2 class="job-title"><a href="single.php?id=<?php echo $row["id"]?>"><?php echo  $row["title"] ?></a></h2>
+						<div class="job-meta">
+							<a class="meta-company" href="single.php?id=<?php echo $row["id"]?>"><?php echo  $row["company_name"] ?></a>
```
Comment (vabrashev) [inline single.php:77]: Is the link for the Company correct?

File: submissions.php [hunk 11]
```diff
@@ -1,82 +1,75 @@
...
+			</ul>
+			<?php
+			$sql = "SELECT *, users.first_name, users.last_name, applications.user_id, applications.id 
+					-- AS APP_ID
+					FROM applications 
+					JOIN users 
+					ON users.id = applications.user_id 
+					JOIN jobs 
+					ON jobs.id = applications.job_id 
+					WHERE applications.job_id = $id
```
Comment (vabrashev) [inline submissions.php:31]: Not sure what will happen if we open the page without id param in the GET

**Review by Neychok — APPROVED** [review]

**Pull request merged**